src/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -22,8 +22,26 @@
  */
 @CommandLineProgramProperties(
         usage = "Takes an input BAM and reference sequence and runs one or more Picard " +
```

Comment:
Still needs to be converted to using  USAGE_SUMMARY and USAGE_DETAILS syntax like the rest